.circleci/config.yml

```diff
@@ -1,11 +1,22 @@
-version: 2.0
+version: 2.1
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - build
+      - "test-arangodb-3_4_8":
```

Comment:
Was the dot notation not working for declaring builds? Like `3.4.8`

Reply:
No, circleci complained about it. Letters, numbers, spaces, dash and underscore are all that's allowed.